EventKit/eventkit-cloud
# bumps gdalutils and django-audit-logging

## Description of Improvement

Updates the gdalutils and django-audit-logging dependencies.

## How to test

1. Build conda artifacts. 
2. Build docker images.
3. Run tests, ensure that the new versions appear in the eventkit image (i.e. conda list). 

## Quality Assurance 

The following items have been updated:
 
## Screenshots

:smile:
